src/control/server/config/server_test.go

```diff
@@ -533,6 +609,49 @@ func TestServerConfig_Validation(t *testing.T) {
 						WithPinnedNumaNode(0),
 				),
 		},
+		"calculated hugepages; emulated bdevs configured": {
```

Comment:
Does "calculated" huge pages refer to no nr_hugepages in the server config?

is it worth a test case for no nr_hugepages specified, with no bdevs in the configuration? Not sure what the expConfig NrHugePages would be though.

Reply:
renamed the test text for clarity and added test case for none specified with no bdevs configured